spring-integration-jms/src/test/java/org/springframework/integration/jms/JmsOutboundGatewayTests.java

```diff
@@ -67,7 +68,7 @@
  *
  * @since 2.2.4
  */
-public class JmsOutboundGatewayTests {
+public class JmsOutboundGatewayTests extends ActiveMQMultiContextTests {
```

Comment:
How does this `extends ActiveMQMultiContextTests` help here?
I don't see any `connectionFactory` reusing below.
Thanks

Reply:
There are a couple of tests in this class that create and destroy CCFs. When there are no connections, the broker stops. Subclassing this keeps the broker running across all the tests in the class.